src/common/actions/auth-store.js

```diff
@@ -270,7 +272,13 @@ async function setShortNamespace(root, discharge, userName) {
     }
     return data;
   } else {
-    const json = await response.json();
+    const text = await response.text();
```

Comment:
Why not just read the JSON with `.json()`? Some responses are not formatted as JSON?

If not `.json()` will throw an error we can catch.